src/many-abci/src/module.rs

```diff
@@ -195,7 +199,9 @@ impl<C: Client + Send + Sync> blockchain::BlockchainModuleBackend for AbciBlockc
         Ok(blockchain::TransactionReturns {
             txn: Transaction {
                 id: TransactionIdentifier { hash: tx_hash },
-                content: None,
+                content_: None,
+                request: None,
+                response: None,
```

Comment:
We should probably fill those out

Reply:
The content field here was not previously filled.  How should we populate these fields otherwise?